multicore_runtime/static_files.py

```diff
@@ -14,18 +14,23 @@
 #
 """Static file serving functionality invoked by wsgi_config.py."""
 
+import datetime
 import httplib
 import logging
 import mimetypes
 import os
 import re
 
+from werkzeug import datastructures
+from werkzeug import http
 from werkzeug import wrappers
 from werkzeug import wsgi
 
 
 def static_app_for_regex_and_files(url_re, files_mapping, upload_re,
-                                   mime_type=None):
+                                   mime_type=None,
+                                   http_headers=None,
+                                   expiration=None):
```

Comment:
I think it would be better if this function accepted a python object, not a string, for expiration. static_app_for_handler() should abstract all specific details of our user interface (app.yaml) from the internals of the system.

In this case I think the "timedelta" python object might be appropriate.